src/components/features/Register/Register.jsx

```diff
@@ -0,0 +1,148 @@
+import axios from 'axios';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+
+import { API_ENDPOINT } from '../../../config/config.js';
+
+import './Register.scss';
+
+const Registration = () => {
+  const [userData, setUserData] = useState({
+    identifier: '',
+    password: '',
+    file: [],
+  });
+
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    // eslint-disable-next-line no-unused-vars
+    reset,
+    trigger,
+  } = useForm();
+
+  const handleSubmitRegistration = (data) => {
+    setUserData(data);
+
+    const handleRegistration = async () => {
+      try {
+        console.log(data, 'async');
+        const response = await axios.post(`${API_ENDPOINT}register`, data);
+        localStorage.setItem('userJwt', response.data.jwt);
```

Comment:
We don't want to do this from the component - we want to move this logic to the auth context - AuthContext should same the jwt token as a state variable, and save it to localStorage (and read it from localStorage if it exists when the app is first started)